doc/md/guides/mysql/descending-indexes.md

```diff
@@ -0,0 +1,364 @@
+---
+id: descending-indexes
+title: Descending Indexes in MySQL
+slug: /guides/mysql/descending-indexes
+---
+
+### What are the descending indexes?
+In general, indexes with ascending or descending order help increase the performance of queries with the “ORDER BY” clause. Descending indexes are indexes where key values are stored in descending order.
+
+#### When are descending indexes helpful?
+In versions prior to MySQL 8.0, scanning an index in reverse order had a very high cost, which resulted in reduced performance for certain queries. 
+Since the release of MySQL version 8.0, users can now create descending indexes, which can be scanned in forward order, thus increasing efficiency of scanning for certain queries with ORDER BY clause.
```

Comment:
```suggestion
Since the release of MySQL version 8.0, users can now create descending indexes, which can be scanned in forward order, thus increasing efficiency of scanning for certain queries with the ORDER BY clause.
```